core/src/main/java/org/apache/iceberg/FileMetadata.java

```diff
@@ -111,7 +111,7 @@ public Builder ofPositionDeletes() {
       return this;
     }
 
-    public Builder ofEqualityDeletes() {
+    public Builder ofEqualityDeletes(int... equalityFieldIds) {
```

Comment:
These IDs will be added to `DeleteFile` metadata in a separate PR. This is just a placeholder for now.

Reply:
The PR for this is #1318.